examples/rac-spectrum-tailwind/README.md

```diff
@@ -0,0 +1,24 @@
+# React Aria Components + Spectrum + Tailwind CSS
+
+A Tailwind CSS template for creating React Aria Components using common Spectrum styles.
+
+## Overview
+
+### Includes
+
+- `Theme`: a preset that includes Spectrum values for colors, scale and more
+- `Plugin`: for easier styling based on React Aria Component's selectors for states
+
+### When to use this
+
+- You need a component that matches an ARIA pattern, but doesn't exist in React Spectrum
+- You are prototyping a component that might be a good fit for a future addition to React Spectrum, but isn't available yet
+
+### When not to use this
+
+- There is an existing React Spectrum component that matches your needs
+- You are trying to override Spectrum guidelines or create a slight variation of an existing React Spectrum component
+
+### Note
+
+This is not a library you can install and consume, but rather a template that you can copy into your own project and maintain yourself. Keep in mind that you are still responsible for ensuring the experiences you create are accessible and meet Spectrum guidelines.
```

Comment:
What rough level of maintenance do you foresee teams having to do? Theoretically there shouldn't be too much since the vars in the preset come from our Provider/Spectrum CSS directly? I guess it would be safer for us to publish this as a package like you mentioned earlier to minimize the chance that people would heavily modify the preset locally.

Reply:
Yeah, I'm leaning towards publishing it as a package. I don't think there would be any maintenance if teams just copied, unless there is a React Spectrum upgrade that changes CSS variables. In that case, if we are publishing a package, we could bump the version. Also, consuming it will just be easier as a package. We could maybe even include a test that asserts that all the variables in the theme exist if the provider is used, so we would catch any breaking changes.